accesspoc/dips/forms.py

```diff
@@ -98,3 +101,38 @@ def save(self, commit=True):
         if commit:
             user.save()
         return user
+
+
+class DublinCoreSettingsForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        super(forms.Form, self).__init__(*args, **kwargs)
+        self.optional_fields_setting = Setting.objects.get(name='enabled_optional_dc_fields')
```

Comment:
I know it's unlikely but you almos want to confirm that `Setting.DoesNotExist` is being raised because that would make the application unusable.

Reply:
If the settings don't exists, it will only raise an error in the settings page and the forms that use them, which I think is okay, as those pages are only shown to admins or editors.